docs/blog/2025-03-19-using-angular-with-rspack.md

```diff
@@ -0,0 +1,166 @@
+---
+title: 'Using Angular with Rspack'
+date: 2025-03-19
+slug: using-angular-with-rspack
+authors: [Colum Ferry]
+tags: [angular, webpack, rspack, nx]
+cover_image: /blog/images/2025-03-19/angular-rspack-logo-small.png
+description: Learn about how and why to use Rspack with Angular thanks to Nx's efforts for supporting Rspack for Angular.
+---
+
+Building applications with [Angular](https://angular.dev) has always been a lesser concern for most Angular developers due to the abstractions that Angular created called `builders`. The underlying implementation details were hidden from the developer who only needed to run either `ng build` or `nx build`.  
+Despite this, most Angular developers knew that it was originally [webpack](https://webpack.js.org/) that was used to build their applications. This was a great solution at the time and it was even possible to extend their builds by leveraging custom webpack configurations and plugins.
+
+Over time, as applications grew in size and complexity, it became clear that the inherit slowness with Webpack build speeds was becoming more and more of an issue for Angular developers.
+
+The Angular Team decided to address this build speed issue by building out a new build pipeline that leveraged [Esbuild](https://esbuild.github.io/).
+
+This succeeded in reducing the build times for Angular applications, however, it made one crucial mistake. It left the existing Angular applications that relied on the Webpack ecosystem behind, with either a difficult migration path or none at all.
```

Comment:
I think this would be worth emphasizing more. On the enterprise articles we have a component for large quotes:

![image](https://github.com/user-attachments/assets/c042ffa3-9baa-44f0-8278-dd09c54484f3)

I think we could use the same, but without the author obviously. Alternatively we can just use a blockquote

Reply:
Both paragraphs or the one about the leaving webpack users behind?

Reply:
Maybe paraphrase and summarize it. Like

> Esbuild brought much-needed performance improvements to Angular builds, but existing Webpack-based applications were left with either a difficult migration path or no clear upgrade strategy.

or ideally even shorter. but something like this